kaldb/src/main/java/com/slack/kaldb/writer/kafka/KaldbKafkaConsumer.java

```diff
@@ -77,6 +77,7 @@ private static Properties makeKafkaConsumerProps(
         "org.apache.kafka.common.serialization.ByteArrayDeserializer");
     // TODO: Does the session timeout matter in assign?
     props.put(ConsumerConfig.SESSION_TIMEOUT_MS_CONFIG, kafkaSessionTimeout);
+    props.put(ConsumerConfig.AUTO_OFFSET_RESET_CONFIG, "none");
```

Comment:
A comment here explaining why this is "none" would be good - just to prevent someone from attempting to change it in the future without realizing the motivation.